README.md

```diff
@@ -1 +1,45 @@
-# Portfolio-Setup_Mobile_Skeleton
\ No newline at end of file
+# Portfolio-Setup_Mobile_Skeleton
+
+> Kicking off developing  my portfolio as a microverse student with Mobile skeleton setup using basic HTML5 tag, CSS3 and applying linters.
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML5
+- CSS3
+- Linters
+- application of font awesome
+
+## Live Demo
+
+[Live Demo Link](https://olusegun21.github.io/Portfolio-Setup_Mobile_Skeleton/)
+
+
+## Author
+
+👤 **Olusegun aka Bosby**
+
+- GitHub: [@olusegun21](https://github.com/olusegun21)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/olusegun-olagunju)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## Acknowledgments
+
+- Microverse
+- Figma
+- My coding partners 
```

Comment:
- [ ] Amazing README file!. Please let's include the **Getting Started** and **License** sections. This will make our README file more descriptive and professional so that a visitor can easily understand how to get started with your project. You may also take a look at this template and later customize it to fit your project.

Reply:
Hi @FrederickMih,

Trust you are well. I will like to appreciate your time given to review my project. However, I will like to seek more clarification probably outside the review process on the comment you made on my project

1. Regarding the license, is it always compulsory to have a license section on the README file when the project is not completed?
2. Although I added the MIT.md file and indicated MIT under license in the README file I do not WHY. also, I saw APP TYPE in the template I used, I will like to seek clarification on APP TYPE. What should one fill in that angular bracket that has APP TYPE? 

Your kind response will be highly appreciated.